ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -122,6 +129,19 @@
       SerdeFeatures.of()
   );
 
+  private static final String PROPS_JAAS_REALM = "KsqlServer-Props";
+  private static final String KSQL_RESOURCE = "ksql-user";
+  private static final String USER_WITH_ACCESS = "harry";
```

Comment:
do we have a test trying out a user with no access? that will make sure that we haven't accidentally just changed it to allow all users 😆

Reply:
We do have tests elsewhere that test auth for pull queries and check the negative case.